packages/terra-clinical-item-view/tests/wdio/item-view-spec.js

```diff
@@ -35,10 +48,19 @@ Terra.describeViewports('Clinical Item View', ['tiny', 'small', 'medium', 'large
 });
 
 Terra.describeViewports('Clinical Item View', ['enormous'], () => {
-  it('with the full example truncated - one truncated', () => {
+  it('with truncation', () => {
     browser.url('/raw/tests/terra-clinical-item-view/clinical-item-view/overflow-displays-item-view');
 
-    Terra.validates.element('with truncation in one column', { selector: '#ItemView-one-wrap' });
-    Terra.validates.element('with truncation in two columns', { selector: '#ItemView-two-wrap' });
+    Terra.validates.element('with truncation in one column', { selector: '#ItemView-one-truncate' });
+    Terra.validates.element('with truncation in two columns', { selector: '#ItemView-two-truncate' });
```

Comment:
Noticed that previously this was not actually pointing to the tests for truncated content. So any of the 'with truncation in one column' or 'with truncation in two columns' snapshots are going to be changed for good reason.

Reply:
Also some of the 'with word wrap' images have changed a bit as well - just the sizing of that image has changed but not the actual word wrap/overflow behavior. I will post recordings in the description to show this.